### PR TITLE
IndependenceDay

### PR DESCRIPTION
**Pull request by moozzyk**
Creating a dedicated SignalR store client

### CODE REVIEW

File: src/Microsoft.AspNet.SignalR.Client.Store/Microsoft.AspNet.SignalR.Client.Store.csproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.AspNet.SignalR.Client</RootNamespace>
-    <AssemblyName>Microsoft.AspNet.SignalR.Client.Store</AssemblyName>
```
Comment (halter73) [inline src/Microsoft.AspNet.SignalR.Client.Store/Microsoft.AspNet.SignalR.Client.Store.csproj]: Here's the original PR that added this project: https://github.com/SignalR/SignalR/pull/3028/

I don't see any discussion as to why we went with multiple dlls despite the downside of never automatically choosing the Websocket transport. A quick check of a few follow up PRs also came up empty.

I think making this a single dll is a good change.

**Comment by halter73** [comment]
:shipit:

**Comment by davidfowl** [comment]
:shipit: this is netcore50 right?

**Comment by moozzyk** [comment]
@davidfowl - this is actually portable-win81+wpa81

File: src/Microsoft.AspNet.SignalR.Client.Store/Microsoft.AspNet.SignalR.Client.Store.csproj (hunk 1 above)
Comment (moozzyk) [inline src/Microsoft.AspNet.SignalR.Client.Store/Microsoft.AspNet.SignalR.Client.Store.csproj]: @halter73 I tried digging but could not find anything on this either...

**Pull request merged**